sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java

```diff
@@ -110,9 +136,7 @@ private JdbcIO() {
    * object used in the {@link PCollection}.
    */
   public interface RowMapper<T> extends Serializable {
-
       T mapRow(ResultSet resultSet);
```

Comment:
Indentation seems off, 4 spaces instead of 2. Consider running https://github.com/google/google-java-format on the PR because there are some other places with incorrect indentation.